pkg/querier/correctness/runner.go

```diff
@@ -105,11 +110,14 @@ func NewRunner(cfg RunnerConfig) (*Runner, error) {
 		Address: cfg.prometheusAddr,
 	}
 	if cfg.userID != "" {
-		apiCfg.RoundTripper = promhttp.RoundTripperFunc(func(req *http.Request) (*http.Response, error) {
-			user.InjectOrgIDIntoHTTPRequest(user.InjectOrgID(context.Background(), cfg.userID), req)
-			return http.DefaultTransport.RoundTrip(req)
-		})
+		apiCfg.RoundTripper = &nethttp.Transport{
```

Comment:
Why the change here? Seems like it's just another wrapper but I don't think anything changed to make it needed?

Reply:
The opentracing http client is a bit weird - you need the call to nethttp.TraceRequest _and_ the nethttp.Transport.